docs/libs/ml/optimization.md

```diff
@@ -160,10 +158,20 @@ mini-batch SGD will be performed.
         <td><strong>RegularizationParameter</strong></td>
         <td>
           <p>
-            The amount of regularization to apply. (Default value:<strong>0</strong>)
+            The amount of regularization to apply. (Default value:<strong>0.0</strong>)
+          </p>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>PredictionFunction</strong></td>
+        <td>
+          <p>
+            Class that provides the prediction function, used to calculate $\hat{y}$ based on the
+            weights and the example features, and the prediction gradient.
+            (Default value: <strong>LinearPrediction</strong>)
```

Comment:
Should I add these in their own table then? i.e. one for prediction functions, one for loss functions etc.